test/unit/api_spec.js

```diff
@@ -2624,6 +2624,17 @@ Caron Broadcasting, Inc., an Ohio corporation (“Lessee”).`)
       await loadingTask.destroy();
     });
 
+    it("gets text content with a rised text", async function () {
+      const loadingTask = getDocument(buildGetDocumentParams("issue16221.pdf"));
+      const pdfDoc = await loadingTask.promise;
+      const pdfPage = await pdfDoc.getPage(1);
+      const { items } = await pdfPage.getTextContent();
+
+      expect(items.map(i => i.str)).toEqual(["Hello ", "World"]);
+
+      await loadingTask.destroy();
+    });
+
```

Comment:
To me this feels like the sort of thing that would be much better tested with a `text` reference-test, rather than a unit-test.

(I've always felt that it can be tempting to just update the expected value in a unit-test, when making future changes, whereas it's slightly more difficult to just ignore a failing ref-test.)